src/main/java/crazypants/enderzoo/entity/EntityEnderminy.java

```diff
@@ -123,7 +123,7 @@ public boolean getCanSpawnHere() {
    */
   private boolean shouldAttackPlayer(EntityPlayer player) {
 
-    ItemStack itemstack = player.inventory.armorInventory[3];
+    ItemStack itemstack = player.inventory.armorInventory.get(3);
```

Comment:
Yeah ill check. do you remember what  slot it was looking for? pants or something? ill check the documentation too